Gemfile

```diff
@@ -230,6 +230,10 @@ platforms :jruby do
   end
 end
 
+group :opf_plugins do
+  gem 'openproject-translations', git:'https://github.com/opf/openproject-translations.git', branch: 'dev'
```

Comment:
I think `dev` is not correct, it should be `release/4.2` while when merging up, it needs to be adapted. We need a way to update it in lockstep with the OP version.